requirements/base.txt

```diff
@@ -1,17 +1,17 @@
 chevron~=0.12
-click~=7.0
-Flask~=1.0.2
+click~=7.1
+Flask~=1.1.2
 #Need to add Schemas latest SDK.
 boto3~=1.14.0, >=1.14.23
-jmespath~=0.9.5
-PyYAML~=5.1
-cookiecutter~=1.6.0
+jmespath~=0.10.0
+PyYAML~=5.3
+cookiecutter~=1.7.2
 aws-sam-translator==1.27.0
 #docker minor version updates can include breaking changes. Auto update micro version only.
 docker~=4.2.0
 dateparser~=0.7
 python-dateutil~=2.6, <2.8.1
 requests==2.23.0
-serverlessrepo==0.1.9
+serverlessrepo==0.1.10
```

Comment:
What are the changes here? Do we need updated integ tests?

Reply:
See the commit message for detail: https://github.com/aws/aws-sam-cli/pull/2305/commits/f250d39e4bc3e678260b2397e0e69edb205a13c2

>  chore: Bump serverlessrepo to 0.1.10
only one change since 0.1.9:
Support LicenseBody and ReadmeBody application metadata properties

Reply:
Oh I see what you mean. Do you know where an existing test example located? thanks!

Reply:
At some point I added integ tests here for this task :( https://github.com/c2tarun/aws-sam-cli/commit/792695851cbf988fede9dd399747f0d00757fcff

It fell off my radar due to some other issues. @aahung please feel free to grab tests from there. But since it was long time ago, please ensure that tests are still valid and correct.

Reply:
Thanks @c2tarun!
Now integration tests are here now.

Note: you might find licensebody is only tested for application creation and readmebody is only tested for application update. It took me a while to find that out. I think this is intended behavior and respect the tests in serverlessrepo:

https://github.com/awslabs/aws-serverlessrepo-python/blob/b08cb944d93519e05087fa51993a778623b69ec0/tests/unit/test_publish.py#L314-L355